### PR TITLE
added a field to add the feedback in Application evaluation dialog box

### PR DESCRIPTION
**Pull request by rajatsajwan**
Targets #2682
### Description
Ability to add feedback in Application evaluation box

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Facing the following error while evaluating new applications, seems like for those there is no comment, a fallback should be added. Please check.
image

**Comment by rajatsajwan** [comment]
Screenshot_20230113_104702
Screenshot_20230113_104734
Screenshot (53)

File: resources/views/hr/application/evaluation-form.blade.php [hunk 1]
```diff
@@ -22,22 +22,30 @@
...
+    </div>
     @else
-        <div class="evaluation-stage" id="evaluationStage1">
-            @include('hr::evaluation.evaluation-form.resume-screening.feeling')
+
+    @php
+    $applicationRoundReview = $applicationRound->applicationRoundReviews->where('review_key', 'feedback')->first();
+    $applicationRoundReviewValue = $applicationRoundReview ? $applicationRoundReview->review_value : '';
+    @endphp
+
```
Comment (P4NK4J) [inline resources/views/hr/application/evaluation-form.blade.php:37]: These queries should be moved either to service or controller at least, blade file should not be determining any database logic.

File: resources/views/hr/application/evaluation-form.blade.php [hunk 2]
```diff
@@ -22,22 +22,30 @@
...
+    <div class="evaluation-stage" id="evaluationStage1">
+        @include('hr::evaluation.evaluation-form.resume-screening.feeling')
+        {{-- @dd($applicationRound); --}}
```
Comment (P4NK4J) [inline resources/views/hr/application/evaluation-form.blade.php:40]: Didn't expect to push commented lines, please do a self review

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left a comment

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**